ts/javascript/vite.ts

```diff
@@ -0,0 +1,55 @@
+import { Executable } from "../executable.ts";
+import { nixRaw } from "../nix.ts";
+import { Package } from "../package.ts";
+import { Plugin } from "../project.ts";
+
+/**
+ * `Plugin` for [vite](https://vitejs.dev/) projects. This will add to your
```

Comment:
Does the markdown link syntax work in your editor or deno doc? It doesn't have any affect in my LSP

Reply:
Yeah, it does work, both in my editor and on doc.deno.land. It's a bit cumbersome to click on it in my editor, since it's a hover, but it's possible. Generally deno seems to both recommend and use markdown. (It's recommended [here](https://docs.deno.com/runtime/manual/references/contributing/style_guide#use-jsdoc-for-exported-symbols) and here's [an example](https://deno.land/std@0.206.0/toml/mod.ts?source=#L6).)